scipy/stats/stats.py

```diff
@@ -346,6 +346,26 @@ def _broadcast_shapes_with_dropped_axis(a, b, axis):
     return shp
 
 
+def _convert_symmetric_p_value(pvalue, root_func, alternative):
+    """
+    Convert a one-sided, symmetric P-value to a one or two-sided p-value. root_func
```

Comment:
```suggestion
    Convert a one-sided p-value from a symmetric distribution to a one or two-sided p-value.
```
The first line of a docstring should fit on just one line and shouldn't refer to parameters. See https://numpy.org/devdocs/docs/howto_document.html#docstring-standard

Reply:
Please let me resolve comments so I can confirm that they are actually resolved.